tests/components/ring/test_sensor.py

```diff
@@ -1,122 +1,46 @@
-        )
-        mock.get(
-            "https://api.ring.com/clients_api/chimes/999999/health",
-            text=load_fixture("ring_chime_health_attrs.json"),
-        )
+    assert downstairs_wifi_signal_strength_state is not None
+    assert downstairs_wifi_signal_strength_state.state == "-39"
 
-        with mock_storage(), patch("homeassistant.components.ring.PLATFORMS", []):
-            run_coroutine_threadsafe(
-                base_ring.async_setup(self.hass, VALID_CONFIG), self.hass.loop
-            ).result()
-            run_coroutine_threadsafe(
-                self.hass.async_block_till_done(), self.hass.loop
-            ).result()
-            run_coroutine_threadsafe(
-                ring.async_setup_entry(self.hass, None, self.add_entities),
-                self.hass.loop,
-            ).result()
+    front_door_wifi_signal_category_state = hass.states.get(
+        "sensor.front_door_wifi_signal_category"
+    )
+    assert front_door_wifi_signal_category_state is not None
+    assert front_door_wifi_signal_category_state.state == "good"
 
-        for device in self.DEVICES:
-            # Mimick add to hass
-            device.hass = self.hass
-            run_coroutine_threadsafe(
-                device.async_added_to_hass(), self.hass.loop,
-            ).result()
-
-            # Entity update data from ring data
-            device.update()
-            if device.name == "Front Battery":
-                expected_icon = icon_for_battery_level(
-                    battery_level=int(device.state), charging=False
-                )
-                assert device.icon == expected_icon
-                assert 80 == device.state
-            if device.name == "Front Door Battery":
-                assert 100 == device.state
-            if device.name == "Downstairs Volume":
-                assert 2 == device.state
-                assert "ring_mock_wifi" == device.device_state_attributes["wifi_name"]
-                assert "mdi:bell-ring" == device.icon
-            if device.name == "Front Door Last Activity":
-                assert not device.device_state_attributes["answered"]
-                assert "America/New_York" == device.device_state_attributes["timezone"]
-
-            if device.name == "Downstairs WiFi Signal Strength":
-                assert -39 == device.state
-
-            if device.name == "Front Door WiFi Signal Category":
-                assert "good" == device.state
-
-            if device.name == "Front Door WiFi Signal Strength":
-                assert -58 == device.state
-
-            assert device.entity_picture is None
-            assert ATTRIBUTION == device.device_state_attributes["attribution"]
-            assert not device.should_poll
+    front_door_wifi_signal_strength_state = hass.states.get(
+        "sensor.front_door_wifi_signal_strength"
+    )
+    assert front_door_wifi_signal_strength_state is not None
+    assert front_door_wifi_signal_strength_state.state == "-58"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4821145)